Failed to load plugin

Stack trace:

```
Error: Cannot find module 'react'
Require stack:
- /Users/surendra/Library/Application Support/Insomnia/plugins/insomnia-plugin-free-sync/index.js
- /Applications/Insomnia.app/Contents/Resources/app.asar/index.html
    at Module._resolveFilename (node:internal/modules/cjs/loader:963:15)
    at o._resolveFilename (node:electron/js2c/renderer_init:2:3833)
    at Module._load (node:internal/modules/cjs/loader:811:27)
    at f._load (node:electron/js2c/asar_bundle:2:13328)
    at o._load (node:electron/js2c/renderer_init:2:3109)
    at Module.require (node:internal/modules/cjs/loader:1035:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/surendra/Library/Application Support/Insomnia/plugins/insomnia-plugin-free-sync/index.js:201:29)
    at Object.<anonymous> (/Users/surendra/Library/Application Support/Insomnia/plugins/insomnia-plugin-free-sync/index.js:408:3)
    at Module._compile (node:internal/modules/cjs/loader:1141:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1196:10)
    at Module.load (node:internal/modules/cjs/loader:1011:32)
    at Module._load (node:internal/modules/cjs/loader:846:12)
    at f._load (node:electron/js2c/asar_bundle:2:13328)
    at o._load (node:electron/js2c/renderer_init:2:3109)
    at Module.require (node:internal/modules/cjs/loader:1035:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at z9 (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index.7cb7367d.js:67:75235)
    at ND (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index.7cb7367d.js:67:75885)
    at async f (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index.7cb7367d.js:176:880)
```

Version:
Insomnia 2022.7.0
Build date: 12/15/2022
OS: Darwin arm64 21.5.0
Electron: 22.0.0
Node: 16.17.1
V8: 10.8.168.20-electron.0
Architecture: arm64

Same issue on Win11 Insomnia v2022.7.5
`Error: Cannot find module 'react'
Require stack:
- C:\Users\zzyyz\AppData\Roaming\Insomnia\plugins\insomnia-plugin-free-sync\index.js
- C:\Users\zzyyz\AppData\Local\insomnia\app-2022.7.5\resources\app.asar\index.html
    at Module._resolveFilename (node:internal/modules/cjs/loader:963:15)
    at o._resolveFilename (node:electron/js2c/renderer_init:2:3833)
    at Module._load (node:internal/modules/cjs/loader:811:27)
    at f._load (node:electron/js2c/asar_bundle:2:13328)
    at o._load (node:electron/js2c/renderer_init:2:3109)
    at Module.require (node:internal/modules/cjs/loader:1035:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\Users\zzyyz\AppData\Roaming\Insomnia\plugins\insomnia-plugin-free-sync\index.js:201:29)
    at Object.<anonymous> (C:\Users\zzyyz\AppData\Roaming\Insomnia\plugins\insomnia-plugin-free-sync\index.js:408:3)
    at Module._compile (node:internal/modules/cjs/loader:1141:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1196:10)
    at Module.load (node:internal/modules/cjs/loader:1011:32)
    at Module._load (node:internal/modules/cjs/loader:846:12)
    at f._load (node:electron/js2c/asar_bundle:2:13328)
    at o._load (node:electron/js2c/renderer_init:2:3109)
    at Module.require (node:internal/modules/cjs/loader:1035:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at A7 (file:///C:/Users/zzyyz/AppData/Local/insomnia/app-2022.7.5/resources/app.asar/index.aff207ac.js:67:75119)
    at II (file:///C:/Users/zzyyz/AppData/Local/insomnia/app-2022.7.5/resources/app.asar/index.aff207ac.js:67:75769)
    at async f (file:///C:/Users/zzyyz/AppData/Local/insomnia/app-2022.7.5/resources/app.asar/index.aff207ac.js:176:880)
    at async onSubmit (file:///C:/Users/zzyyz/AppData/Local/insomnia/app-2022.7.5/resources/app.asar/index.aff207ac.js:178:530)`

--- a/npm/generate-package.js
+++ b/npm/generate-package.js
@@ -46,6 +46,7 @@ async function executeAsync() {
     repository: dataObj.repository,
     bugs: dataObj.bugs,
     insomnia: dataObj.insomnia,
+    dependencies: dataObj.dependencies,
   }
 
   const resultJson = JSON.stringify(resultObj, null, 2)

